### PR TITLE
MMS tutorial

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
ticket: CVS-115237

### CODE REVIEW

File: notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb [hunk 1]
```diff
@@ -0,0 +1,599 @@
+{
```
Comment (igor-davidyuk) [inline notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: <h2>Prerequisites</h2><p>Should work better as a section title, as packages are both installed and imported in this section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (igor-davidyuk) [inline notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    import openvino
</pre><p><code>import openvino as ov</code> is a conventional  way</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (igor-davidyuk) [inline notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: <p>Does not sound right for me. May be we change it to <code>Do inference</code>? Or <code>Original model inference</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

File: notebooks/255-mms-massively-multilingual-speech/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# MMS: Scaling Speech Technology to 1000+ languages with OpenVINO™
```
Comment (igor-davidyuk) [inline notebooks/255-mms-massively-multilingual-speech/README.md:1]: Does it work in Google Colab / Binder? If so please add badges

**Review by sbalandi — APPROVED** [review]
LGTM, but I would specify transformers version , because I had old and it caused errors in cell with Wav2Vec2ForCTC, AutoProcessor

File: notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb (hunk 1 above)
Comment (aleksandr-mokrov) [reply notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: Changed

Comment (aleksandr-mokrov) [reply notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: Changed

File: notebooks/255-mms-massively-multilingual-speech/README.md (hunk 2 above)
Comment (aleksandr-mokrov) [reply notebooks/255-mms-massively-multilingual-speech/README.md:1]: Unfortunately, there is not enough RAM

File: notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb (hunk 1 above)
Comment (aleksandr-mokrov) [reply notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: Paraphrased

**Review by igor-davidyuk — APPROVED** [review]
looks good

File: notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb (hunk 1 above)
Comment (eaidova) [inline notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    compiled_asr_model = core.compile_model(asr_model_xml_path, device_name='CPU')
</pre><p>please add device selection</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (eaidova) [inline notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: <p>could you please add oportunity to run models on user provided</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (aleksandr-mokrov) [reply notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: added

Comment (aleksandr-mokrov) [reply notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: Gradio demo added for a user audio pipeline

File: notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
```
Comment (eaidova) [inline notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.        if not asr_model_xml_path.exists():
</pre><p>Does asr_model.load_adapter need only for conversion?</p><p> </p><p>If yes, then probably we can move adapter loading under condition that model is not converted yet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (eaidova) [inline notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    def regognize_audio(compiled_asr_model, src_audio):
</pre><p>regognize -&gt; recognize</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

**Comment by eaidova** [comment]
@aleksandr-mokrov we avoid storing images inside repo, please replace mms_sceme.png with link on image uploaded on github

File: notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb (hunk 3 above)
Comment (eaidova) [reply notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: <p>also it is good for demonstration result provide also reference and add audio player with input audio that user can check quality of the result </p>

Comment (aleksandr-mokrov) [reply notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: Moved

Comment (aleksandr-mokrov) [reply notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb:1]: Done

**Review by eaidova — APPROVED** [review]

**Pull request merged**